src/server/testUtils.js

```diff
@@ -23,10 +23,10 @@ export const useDatabase = () => {
   beforeEach(async function () {
     knex = connect('test')
     await knex.migrate.latest()
+    await truncateAll(knex)
   })
 
   afterEach(async function () {
-    await truncateAll(knex)
```

Comment:
No because it's nice to be able to check test database to see if everything is OK after the test.